WordPress/gutenberg
# Fix: The Copy button of the PostURL shouldn't be placed in the slug input

## What?

Fixes #63700

## Why?

As a user, one would expect the Copy button to copy the value inside the input field, because that's what the UI shows

## How?

Change the first parameter in the `useCopyToClipboard` hook to `/ + postSlug` from `permalink`

## Testing Instructions
- Create a new post 
- From settings change permalink to `Post Name`
- Now go back to the post and on the right panel click on the value part of `Link`
- Copy the value of the slug by pressing the copy button 
- Verify that the value now copied is the slug and not the link to the post 






### Testing Instructions for Keyboard
Same as above 

## Screenshots or screencast 


https://github.com/user-attachments/assets/bdd0b1b2-6f80-4f80-a0ab-0ed2796a43dc
